queries/elm/folds.scm

```diff
@@ -0,0 +1,13 @@
+((function_call_expr) @_fn
+  (#not-has-parent? @_fn parenthesized_expr)) @fold
```

Comment:
Out of curiosity, why no `parenthesized_expr` parents here?

Reply:
Elm-format will place the closing paren on the following line. So with just the function folded it looks like
```
+ fold
    )
```
This seemed like it was worth the extra work to remove that case and just fold on the `parenthesized_expr`.